index.html

```diff
@@ -1468,48 +1468,42 @@ <h3 property="schema:name">Normative References</h3>
                         ><cite
-                          >Hypertext Transfer Protocol (HTTP/1.1): Message
-                          Syntax and Routing</cite
+                          >Ambiguity of Uppercase vs Lowercase in RFC 2119 Key
+                          Words</cite
                         ></a
-                      >. R. Fielding, Ed.; J. Reschke, Ed.. IETF. June 2014.
-                      Proposed Standard. URL:
-                      <a href="https://httpwg.org/specs/rfc7230.html"
-                        >https://httpwg.org/specs/rfc7230.html</a
+                      >. B. Leiba. IETF. May 2017. Best Current Practice. URL:
+                      <a href="https://datatracker.ietf.org/doc/html/rfc8174"
+                        >https://datatracker.ietf.org/doc/html/rfc8174</a
                       >
                     </dd>
-                    <dt id="bib-rfc7231">[RFC7231]</dt>
+                    <dt id="bib-rfc9110">[RFC9110]</dt>
                     <dd>
                       <a
-                        href="https://httpwg.org/specs/rfc7231.html"
+                        href="https://www.rfc-editor.org/rfc/rfc9110"
                         rel="cito:citesAsAuthority"
-                        ><cite
-                          >Hypertext Transfer Protocol (HTTP/1.1): Semantics and
-                          Content</cite
-                        ></a
-                      >. R. Fielding, Ed.; J. Reschke, Ed.. IETF. June 2014.
-                      Proposed Standard. URL:
-                      <a href="https://httpwg.org/specs/rfc7231.html"
-                        >https://httpwg.org/specs/rfc7231.html</a
+                        ><cite>HTTP Semantics</cite></a
+                      >. R. Fielding, Ed.; M. Nottingham, Ed.; J. Reschke, Ed.. IETF. June 2022.
+                      Internet Standard. URL:
+                      <a href="https://www.rfc-editor.org/rfc/rfc9110"
+                        >https://www.rfc-editor.org/rfc/rfc9110</a
                       >
                     </dd>
-                    <dt id="bib-rfc8174">[RFC8174]</dt>
+                    <dt id="bib-rfc9112">[RFC9112]</dt>
                     <dd>
                       <a
-                        href="https://datatracker.ietf.org/doc/html/rfc8174"
+                        href="https://www.rfc-editor.org/rfc/rfc9112"
                         rel="cito:citesAsAuthority"
-                        ><cite
-                          >Ambiguity of Uppercase vs Lowercase in RFC 2119 Key
-                          Words</cite
-                        ></a
-                      >. B. Leiba. IETF. May 2017. Best Current Practice. URL:
-                      <a href="https://datatracker.ietf.org/doc/html/rfc8174"
-                        >https://datatracker.ietf.org/doc/html/rfc8174</a
+                        ><cite>HTTP/1.1</cite></a
+                      >. R. Fielding, Ed.; M. Nottingham, Ed.; J. Reschke, Ed.. IETF. June 2022.
```

Comment:
There's definitely a punctuation error here, but I'm not sure what's appropriate for this citation format (for which I have not been able to locate documentation). `Ed.. IETF.` should become either `Ed. IETF.` (as suggested here) or `Ed., IETF.`
```suggestion
                      >. R. Fielding, Ed.; M. Nottingham, Ed.; J. Reschke, Ed. IETF. June 2022.
```